doc/source/project/organization.rst

```diff
@@ -106,12 +106,13 @@ Pull requests changing or clarifying the procedures governing this repository:
   *-1* binding votes.
 - Should be titled with the prefix *[PROCEDURES]* and tagged with
   the *procedures* tag in Github.
-- Should not be modified once open. If changes are needed, the pull request
-  should be closed, re-opened with modifications, and votes reset.
 - Should be restricted to just modifying the procedures and generally should not
   contain code modifications.
 - If the pull request adds or removes *committers*, there must be a separate
   pull request for each person added or removed.
+- Amendments may be made to the pull request by pushing additional changesets
+  but doing so restarts the moratorium and invalidates any votes which have
```

Comment:
Also unnecessary, given some votes A and B where A votes count and B don't, if we say "we invalidate any votes which have been cast", since none of the B votes are binding, and we only count the A votes, then specifying +1 votes here doesn't change the math of whether or not a PR gets merged..